<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptureDialog</class>
 <widget class="QDialog" name="CaptureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="configLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Device:</string>
       </property>
       <property name="buddy">
        <cstring>cb_device</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cb_device"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Limit:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="sb_limit">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100000000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_limitType">
         <item>
          <property name="text">
           <string>Packets</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>KB</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="ck_timeout">
       <property name="text">
        <string>Timeout:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="sb_timeout">
       <property name="suffix">
        <string> ms</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000000</number>
       </property>
       <property name="value">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_filter"/>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="ck_promiscuous">
       <property name="text">
        <string>Promiscuous</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pb_beginCapture">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Begin Capture</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="lb_packets">
       <property name="text">
        <string>0 Packets</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lb_kilobytes">
       <property name="text">
        <string>0.0 KB</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pb_stopCapture">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop Capture</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_device</tabstop>
  <tabstop>pb_beginCapture</tabstop>
  <tabstop>pb_stopCapture</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
